src/init/index.js

```diff
@@ -13,6 +14,9 @@ module.exports = function init() {
   return prompt().then(result => {
     return configFile
       .writeConfig('.all-contributorsrc', result.config)
+      .then(() => {
+        check(result.contributorFile)
```

Comment:
[minor] this function may be better represented by a name like `ensureFileExists`